db/init/init.sql

```diff
@@ -16,6 +16,8 @@ create table stored_ingredient
 (
   user_id int,
   ingredient_id int,
+  quantity int,
+  unit string,
```

Comment:
I think enumerating this instead of a string might be a better idea.

For example, 0 -> unit, 1 -> mL/L, 2 -> g/Kg